openmeetings-web/src/main/webapp/css/theme_om/jquery-ui.css

```diff
@@ -595,8 +595,8 @@ button.ui-button::-moz-focus-inner {
 }
 .ui-dialog .ui-dialog-titlebar-close {
 	position: absolute;
-	right: .3em;
-	top: 50%;
+	right: 10px;
```

Comment:
This positions it almost the same, but obviously different. Cause "top 50%" just doesn't really work if it's outside of the titlebar.
But I could also not find any other dialog using this CSS. I think there is no Dialog left I think using this.